package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scripts": {
+    "lerna": "lerna",
+    "postinstall": "npm run lernabootstrap && npm run lernainstall",
+    "lernainstall": "lerna exec --no-bail --npm-ci-mode -- npm run postinstall",
+    "lernabootstrap": "lerna bootstrap --npm-ci-mode --ignore-scripts",
+    "compile": "lerna run build"
+  },
+  "devDependencies": {
+    "@theia/plugin": "^0.4.0-next.8c668434",
+    "@theia/plugin-packager": "0.0.1-1539891078",
+    "@types/node": "^10.12.10",
+    "lerna": "^3.4.3",
+    "rimraf": "2.6.2",
+    "typescript": "^3.1.6",
+    "vscode": "^1.1.21"
+  },
+  "dependencies": {
+    "@jsonforms/core": "^2.0.12-rc.4",
+    "npm": "^6.4.1",
```

Comment:
Where is `simple-git` used? `npm` seems to be a weird dependency, when do we need it?

Reply:
`simple-git` is currently used for getting the example and the seed project from github:
https://github.com/eclipsesource/jsonforms-tooling/pull/3/files#diff-ffb680330d5bd20f9eb2d36c5d68ecc0R33

Actually it should be in the pull request #3 

This will be replaced as soon as yeoman is working. I'm not sure about `npm`. I think viet used it for the `npm install` command that is used here (but I'm not sure if that is needed):
https://github.com/eclipsesource/jsonforms-tooling/pull/3/files#diff-ffb680330d5bd20f9eb2d36c5d68ecc0R39

Reply:
Ok, I see. Let's remove `npm` then (but please re-test the changes, just to be 100% sure)

Reply:
Removed it and the `npm` command still works